### DISCUSSION TITLE
ELBO scaling

### DISCUSSION THREAD

**Discussion by InfProbSciX** [cornellius-gp/gpytorch#1799] [General]
I noticed that for a gplvm, the numerical value of the elbo (VariationalELBO) doesn't change much at all w.r.t. the number of data points. Is the ELBO scaled internally so that it is the mean across data?

**Comment by wjmaddox** [comment]
Yes, in general it's meaned across the data. See the type of scaling in https://github.com/cornellius-gp/gpytorch/blob/fc2053b0fc00517880fbc11adc7f5802242eec6a/gpytorch/mlls/_approximate_mll.py#L57

**Comment by InfProbSciX** [comment]
Thank you - why is this the case? Also, would you mind a PR from me adding this detail to the documentation at https://docs.gpytorch.ai/en/latest/_modules/gpytorch/mlls/variational_elbo.html?

**Reply by gpleiss** [reply]
A PR would be great!

**Comment by wjmaddox** [comment]
Sure. Averaging across data points is also done for the MLL and I believe it's to keep everything on the same scale as NN losses in pytorch (which are also traditionally averaged across the batch) with the end goal being easy interpretability of torch.optim learning rates, etc.
